purefa/datadog_checks/purefa/metrics.py

```diff
@@ -1,39 +1,69 @@
 METRIC_MAP = [
     {
+        # shared metrics
+        "purefa_info": {"name": "info"},
```

Comment:
If you're only defining a `name` key, you can use the shorthand:
```python
"purefa_info": "info",
```

In other words, convert all your dicts to strings

Reply:
I would prefer to keep it this way as I was having issues in the past with default_config when using the shorthand.